integration-tests/mongodb-panache/src/main/java/io/quarkus/it/mongodb/panache/duplicateids/TestDuplicateIdEntity.java

```diff
@@ -0,0 +1,21 @@
+package io.quarkus.it.mongodb.panache.duplicateids;
+
+import org.bson.codecs.pojo.annotations.BsonProperty;
+import org.bson.types.ObjectId;
+
+import io.quarkus.mongodb.panache.PanacheMongoEntityBase;
+
+public class TestDuplicateIdEntity extends PanacheMongoEntityBase {
+
+    @BsonProperty(value = "_id")
```

Comment:
This should be `BsonId`.  The driver does use that specific property for this and that and shouldn't be lost.